CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 [1]: https://pypi.org/project/google-cloud-core/#history
 
+### [2.1.1](https://www.github.com/googleapis/python-cloud-core/compare/v2.1.0...v2.1.1) (2021-11-09)
+
+
+### Bug Fixes
+
+* add mypy checking + 'py.typed' files ([#149](https://www.github.com/googleapis/python-cloud-core/issues/149)) ([f20ef60](https://www.github.com/googleapis/python-cloud-core/commit/f20ef60abaff13520d2e7b9719741746bd0d1c7b))
```

Comment:
`add mypy checking + 'py.typed' files (#149) (f20ef60)`

This has the potential to break some people's workflows. I think this should release as 2.2.0 rather than 2.1.1

Reply:
Should I push an empty PR with `Release-As: 2.2.0` in the commit message, and then we edit the changelog manually?